modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionListener.java

```diff
@@ -465,4 +466,15 @@ void handleBuildIndexCommand(BuildIndexCommand cmd, long commandIndex, long comm
             );
         }
     }
+
+    private static <T extends Comparable<T>> void updateTrackerWithIgnoreTrackerClosedException(
```

Comment:
I suggest renaming it to `updateTrackerIgnoringTrackerClosedException`

Reply:
Rename it.